packages/utilities/psammead-calendars/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@bbc/psammead-calendars",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "module": "esm/index.js",
+  "sideEffects": false,
+  "description": "This package provides logic for formatting alternative (non-Gregorian) calendars",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-calendars"
+  },
+  "author": {
+    "name": "Psammead Maintainers",
+    "email": "PsammeadMaintainers@bbc.co.uk"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/bbc/psammead/issues"
+  },
+  "homepage": "https://github.com/BBC-News/psammead/blob/latest/packages/components/psammead-calendars/README.md",
+  "dependencies": {
+    "jalaali-js": "1.1.0",
+    "moment": "^2.24.0"
```

Comment:
Moment can be a peerDependancy :)

Reply:
What's the difference between `peerDependancy` and a normal `dependancy`.

Reply:
Having a peer dependency means that your package needs a dependency that is the same exact dependency as the person installing your package.

So its staying, "to use `psammead-calender` you need to install moment, and we use the same version"